lib/upipe-framers/upipe_h264_framer.c

```diff
@@ -1365,7 +1365,7 @@ static void upipe_h264f_build_flow_def(struct upipe *upipe)
 
     if (upipe_h264f->duration) {
         UBASE_FATAL(upipe, uref_clock_set_latency(flow_def,
-                         upipe_h264f->input_latency + upipe_h264f->duration))
+                         upipe_h264f->input_latency + upipe_h264f->duration * 2))
```

Comment:
It gives me the impression that you are doubling the latency for all streams, not only interlaced.